tests/conftest.py

```diff
@@ -178,3 +178,20 @@ def multi_threading_test_devices():
         "nidaqmx\\tests\\max_config\\nidaqmxMaxConfig.ini to create these devices."
     )
     return None
+
+
+@pytest.fixture(scope="module")
+def test_device(request):
```

Comment:
It could make sense to rename `bridge_device` to `any_bridge_device`, but it looks like only one test (another property test) uses it, so I agree that it would be okay to change that test to use this new fixture.